.github/workflows/nightly-builds.yml

```diff
@@ -86,7 +86,7 @@ jobs:
         # No need to specify the full Scala version. Only the Scala
         # binary version is required and Akka build will set the right
         # full version from it.
-        scalaVersion: ["2.12", "2.13"]
+        scalaVersion: ["2.12", "2.13", "3.1"]
         jdkVersion: ["1.8.0", "1.11", "1.17.0"]
```

Comment:
Does this mean that it will run all 9 combinations? Previously 6.
I think it's overkill to test all combinations.

Reply:
Updated, to use an explicit matrix. Note that PR validations don't run all tests.